docs/usage/scrapy-poet.rst

```diff
@@ -72,3 +72,21 @@ supported one is :class:`scrapy_zyte_api.ExtractFrom`:
 The provider will set the extraction options based on the annotations, so for
 this code ``extractFrom`` will be set to ``httpResponseBody`` in
 ``productOptions``.
+
+Geolocation
+===========
+
+You can specify the geolocation field by adding a
+:class:`scrapy_zyte_api.Geolocation` dependency and annotating it with a
+country code:
+
+.. code-block:: python
+
+    from typing import Annotated
+
+    from scrapy_zyte_api import Geolocation
+
+    @attrs.define
+    class MyPageObject(BasePage):
+        product: Product
+        geolocation: Annotated[Geolocation, "DE"]
```

Comment:
I haven't checked it, but onder if something like this would work:

```py
class _Geolocation:
    pass

def Geolocation(geolocation: str):
    return Annotated[_Geolocation, geolocation]

@attrs.define
class MyPageObject(ProductPage):
    geolocation: Geolocation("DE")
```

Where does it break?

Reply:
`error: Invalid type comment or annotation  [valid-type]` from mypy, though the runtime code works.

Reply:
I see. Let's not handle it here then. This mypy issue seems relevant: https://github.com/python/mypy/issues/13643 (with a suggested workaround).